spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/contributing.adoc

```diff
@@ -0,0 +1,42 @@
+[[contributing]]
+== Contributing
+
+Spring Cloud is released under the non-restrictive Apache 2.0 license,
+and follows a very standard Github development process, using Github
+tracker for issues and merging pull requests into master. If you want
+to contribute even something trivial please do not hesitate, but
+follow the guidelines below.
+
+=== Sign the Contributor License Agreement
+Before we accept a non-trivial patch or pull request we will need you to sign the
+https://support.springsource.com/spring_committer_signup[contributor's agreement].
+Signing the contributor's agreement does not grant anyone commit rights to the main
+repository, but it does mean that we can accept your contributions, and you will get an
+author credit if we do.  Active contributors might be asked to join the core team, and
+given the ability to merge pull requests.
+
+=== Code Conventions and Housekeeping
+None of these is essential for a pull request, but they will all help.  They can also be
+added after the original pull request but before a merge.
+
+* Use the Spring Framework code format conventions. If you use Eclipse
+  you can import formatter settings using the
+  `eclipse-code-formatter.xml` file from the
+  https://github.com/spring-cloud/build/tree/master/eclipse-coding-conventions.xml[Spring
```

Comment:
Frankly, do we really need such sections in the *reference doc*?

And if we do, I think we'll want to have one canonical version (maybe in SCS) and then `include::` it.